src/renderer/widgets/Staking/BondNominate/default/ui/BondNominate.tsx

```diff
@@ -51,7 +51,7 @@ export const BondNominate = () => {
   return (
     <Modal
       isOpen={isModalOpen}
-      size="md"
+      size="fit"
```

Comment:
Please use dynamic value based on step, only step with validators should have 'fit' size, overwise modal will explode if any validation error appears